build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     compile "mrtjp:MrTJPCore:${config.mc.version}-${config.mrtjp.version}:dev"
     compile "codechicken:CodeChickenLib:${config.mc.version}-${config.ccl.version}:dev"
     compile "codechicken:ForgeMultipart:${config.mc.version}-${config.fmp.version}:dev"
+    compile "codechicken:CodeChickenCore:${config.mc.version}-${config.ccc.version}:dev"
```

Comment:
Because you didn't install it in the mods folder. But before you waste your time, I'm letting you know that I've not decided how to implement this gate yet. 




> On Jan 28, 2015, at 8:21 PM, T1ll3 <notifications@github.com> wrote:
> 
> In build.gradle:
> 
> > @@ -69,6 +69,7 @@ dependencies {
> >      compile "mrtjp:MrTJPCore:${config.mc.version}-${config.mrtjp.version}:dev"
> >      compile "codechicken:CodeChickenLib:${config.mc.version}-${config.ccl.version}:dev"
> >      compile "codechicken:ForgeMultipart:${config.mc.version}-${config.fmp.version}:dev"
> > +    compile "codechicken:CodeChickenCore:${config.mc.version}-${config.ccc.version}:dev"
> I get this Error cause CCC 1.0.2.9 is used and this has fixed it. or what am I doing wrong?
> 
> 
> —
> Reply to this email directly or view it on GitHub.
>